src/media/css/footer.styl

```diff
@@ -34,9 +33,9 @@ $support-nav-height = 55px;
             list-style: none;
         }
         p {
-            float: left;
+            flex-grow: 1;
             font-size: 16px;
-            padding: 0 0 0 1em;
+            padding: 0 20px;
             width: 60%;
```

Comment:
Just tried removing this and it makes the extra links too narrow. Seems weird but it appears to help the layout so works for me.